napari/components/_viewer_key_bindings.py

```diff
@@ -134,40 +134,40 @@ def remove_all_layers(viewer):
 remove_all_layers.__doc__ = trans._("Remove all layers.")
 
 
-@ViewerModel.bind_key('Up')
-def select_layer_above(viewer):
-    """Select layer above."""
-    viewer.layers.select_next()
+# @ViewerModel.bind_key('Up')
+# def select_layer_above(viewer):
+#     """Select layer above."""
+#     viewer.layers.select_next()
 
 
-select_layer_above.__doc__ = trans._("Select layer above.")
+# select_layer_above.__doc__ = trans._("Select layer above.")
 
 
-@ViewerModel.bind_key('Down')
-def select_layer_below(viewer):
-    """Select layer below."""
-    viewer.layers.select_previous()
+# @ViewerModel.bind_key('Down')
+# def select_layer_below(viewer):
+#     """Select layer below."""
+#     viewer.layers.select_previous()
 
 
-select_layer_below.__doc__ = trans._("Select layer below.")
+# select_layer_below.__doc__ = trans._("Select layer below.")
 
 
-@ViewerModel.bind_key('Shift-Up')
-def also_select_layer_above(viewer):
-    """Also select layer above."""
-    viewer.layers.select_next(shift=True)
+# @ViewerModel.bind_key('Shift-Up')
+# def also_select_layer_above(viewer):
+#     """Also select layer above."""
+#     viewer.layers.select_next(shift=True)
 
 
-also_select_layer_above.__doc__ = trans._("Also select layer above.")
+# also_select_layer_above.__doc__ = trans._("Also select layer above.")
 
 
-@ViewerModel.bind_key('Shift-Down')
-def also_select_layer_below(viewer):
-    """Also select layer below."""
-    viewer.layers.select_previous(shift=True)
+# @ViewerModel.bind_key('Shift-Down')
+# def also_select_layer_below(viewer):
+#     """Also select layer below."""
+#     viewer.layers.select_previous(shift=True)
 
 
-also_select_layer_below.__doc__ = trans._("Also select layer below.")
+# also_select_layer_below.__doc__ = trans._("Also select layer below.")
```

Comment:
Forgot to uncomment code ?

Reply:
oh right, thanks for reminding me! Yeah we need to discuss this.  All of these keybindings are [provided by default by the QItemView](https://doc.qt.io/qt-5/qabstractitemview.html#details).  We likely originally had all this there because we implemented the LayerList from scratch, but now, technically, we don't need any of it anymore.  _However_: if we want, we can keep it all here and probably add logic to disable it on the QtView ... something to discuss

Reply:
Widgets that provide `motion` shortcuts by default are usually generic and general enough, that we should not allow for customization of shortcuts, plus less code to maintain, so I would go with removing them.

Reply:
the main issue here is the scope/context of the shortcuts.  The behavior that everyone is currently used (for better or worse) to is that up/down will work even if the layerlist is not focused.  For instance, press "add new points" button a few times, and then use up/down (it changes the layer).  If we use the builtins, then we either need to decide we don't want that behavior (consciously) or make sure to pass certain keypresses from the window down to the layerlist

Reply:
i think probably wise to drop, i like the global scope, but probably not worth the custom code/ complexity. if we had been there from the beginning we never would have added

Reply:
great. I'd like let Qt handle the viewer motion shortcuts too, as @goanpeca preferred.  we can address desired behavior via Qt in another PR

Reply:
I removed all the unneeded bindings, to get the global scope back, I _think_ we just need to pass keyPressEvent from qt_main_window to qtlayerlist (@goanpeca, does that sound right?)

Reply:
>(@goanpeca, does that sound right?)

It does :)